index.html

```diff
@@ -165,14 +171,15 @@ <h2 id="eight">Multi-Post<br>Stories</h2>
 
 <!-- ─── About Me Section ─── -->    
 
-    <section class="aboutme">
-        <div class="text">
+    <section id="aboutme">
+        <div id="block1">
+        <div class="abt-text">
             <h2>About<br>Myself</h2>
             <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
         <div class=links>
-            <button type="button" class="button">LET’S CONNECT</button>
-            <ul class="media-links">
+            <button type="button" class="button" id="abt-btn">LET’S CONNECT</button>
+            <ul class="media-links" id="abt-links">
```

Comment:
did not find a solution for that issue, because it seems like it is the background color of the whole website, not only the about me page. As you can see it is the same grey color on the background of the work section and the main page as well.